eng/pipelines/templates/jobs/verify-installers.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       ${{ insert }}: ${{ parameters.Variables }}
       BaseUrl: http://127.0.0.1:8080
 
-    timeoutInMinutes: 10
+    timeoutInMinutes: 15
```

Comment:
Just wondering - I have seen one of the verify installer legs time out from time to time, it this to address that?

Reply:
It is. The 1espt work has added tasks which means we're more likely to hit that timeout now.